sdk/deviceregistry/azure-resourcemanager-deviceregistry/src/test/java/com/azure/resourcemanager/deviceregistry/DeviceRegistryManagerTests.java

```diff
@@ -44,66 +36,28 @@ public void beforeTest() {
             .authenticate(credential, profile)
             .withDefaultSubscription();
 
-        String registrationState
-                = resourceManager.providers().getByName("Microsoft.DeviceRegistry").registrationState();
-        if (!"Registered".equals(registrationState) && !"registering".equals(registrationState)) {
-            registrationState = resourceManager.providers().register("Microsoft.DeviceRegistry").registrationState();
-        }
-        while (!"Registered".equals(registrationState) && !"registering".equals(registrationState)) {
-            try {
-                Thread.sleep(5000);
-            } catch (InterruptedException e) {
-                throw new RuntimeException(e);
-            }
-            registrationState = resourceManager.providers().getByName("Microsoft.DeviceRegistry").registrationState();
-        }
-
         deviceRegistryManager = DeviceRegistryManager.configure()
             .withPolicy(new ProviderRegistrationPolicy(resourceManager))
             .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
             .authenticate(credential, profile);
-
-        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
-        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
-        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
-        if (testEnv) {
-            resourceGroupName = testResourceGroup;
-        } else {
-            resourceManager.resourceGroups().define(resourceGroupName).withRegion(REGION).create();
-        }
-    }
-
-    @Override
-    protected void afterTest() {
-        if (!testEnv) {
-            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
-        }
     }
```

Comment:
Why remove this?

They are still needed for local run.

Reply:
This test case does not create any `ResourceGroup` for local runs, so I think this method can be removed